structs.h

```diff
@@ -1314,21 +1321,21 @@ typedef struct _SNETUIDATA {
 	int size;
 	int uiflags;
 	HWND parentwindow;
-	void(* artcallback)();
-	void(* authcallback)();
-	void(* createcallback)();
-	void(* drawdesccallback)();
-	void(* selectedcallback)();
-	void(* messageboxcallback)();
-	void(* soundcallback)();
-	void(* statuscallback)();
-	void(* getdatacallback)();
-	void(* categorycallback)();
-	void(* categorylistcallback)();
-	void(* newaccountcallback)();
-	void(* profilecallback)();
+	void (*artcallback)();

```

Comment:
A bit of reformatting happening here also, please keep to the format defined by .clang-format.

Reply:
Looks like this formatting was done by .clang-format... was it wrong in the first place or is my formatter screwed up?

Reply:
Oh your right, it's not been formatted properly. If it's not a hassle then leave it out since it would be better to fix it upstream in a relevant commit.

Reply:
Done